src/containers/MyNdla/MyNdlaLayout.tsx

```diff
@@ -146,12 +148,21 @@ const StyledDialogBody = styled(DialogBody, {
 
 const MyNdlaLayout = () => {
   const { t } = useTranslation();
-  const { user, examLock, authenticated } = useContext(AuthContext);
+  const { user, examLock, authenticated, authContextLoaded } = useContext(AuthContext);
   const { updatePersonalData, loading: updateLoading } = useUpdatePersonalData();
   const toast = useToast();
   const location = useLocation();
   const [isOpen, setIsOpen] = useState(false);
 
+  const loginlocation = `/login?state=${routes.myNdla.root}`;
+
+  useEffect(() => {
+    const autologin = getCookie(AUTOLOGIN_COOKIE, document.cookie);
+    if (window.location && autologin && !authenticated && authContextLoaded) {
+      window.location.replace(loginlocation);
+    }
+  }, [loginlocation, authenticated, authContextLoaded]);
+
```

Comment:
Burde dette være en del av `PrivateRoute` istedenfor?

Reply:
Det kunne det vore, men då hadde det ikkje blitt trigga på /minndla.

Reply:
ahaaaa, for `/minndla` er ikke bak login.
Vil denne autologin-saken fungere på sider som er bak login også?

Reply:
Nei for då sendes du til login automatisk. Det som er nytt i disse tilfella er login_hint som gjør at du ikkje trenger å godkjenne innlogging hos feide dersom du har en aktiv sesjon.